recipes/aeppl/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set version = "0.0.6" %}
+
+package:
+  name: aeppl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/aeppl/aeppl-{{ version }}.tar.gz
+  sha256: b0162e21c86ace1c698d5104aac003c994528ce46cd50f643b5a86ed6852dc2d
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
```

`setuptools` isn't needed at run time, just for `setup.py`. Also need a Python version pin for noarch:
```suggestion
    - python >=3.6
    - setuptools
  run:
    - python >=3.6
```